src/components/Order/OrderMain/OrderState/OrderFiatModals/KYCModalTier0/KYCModalTier0.jsx

```diff
@@ -168,24 +168,33 @@ class KYCModal extends Component {
             </h5>
           </div>
 
-          <div className="modal-body">
+            <div className="modal-body">
+            {this.props.kyc.identity_token && (
+                <div>
+                    <h2>{t('order.fiat.kyc.1')}</h2>
+                    <iframe src={`https://ui.idenfy.com/?iframe=true&authToken=${this.props.kyc.identity_token}`} width="100%" height="600" allow="camera" frameborder="0"></iframe>
+                </div>
+            )}
+
             <form onSubmit={this.handleSubmit}>
+                {/*
               {this.props.kyc.id_document_status !== 'APPROVED' && (
                 <div>
                   <h2>{t('order.fiat.kyc.1')}</h2>
                   <small>{t('order.fiat.kyc.11')}</small>
                   <input type="file" name="governmentID" id="governmentID" onChange={this.handleInputChange} accept="image/*" />
                 </div>
               )}
+                */}
 
               {this.props.kyc.residence_document_status !== 'APPROVED' && (
                 <div>
                   <h2>{t('order.fiat.kyc.2')}</h2>
                   <small>
-                    {t('order.fiat.kyc.21')}
+                      <div dangerouslySetInnerHTML={{__html: t('order.fiat.kyc.21')}} />
                   </small>
                   <small>
-                    {t('order.fiat.kyc.22')}
+                      <div dangerouslySetInnerHTML={{__html: t('order.fiat.kyc.22')}} />
```

Comment:
this was a bug (html from translations was not working)

Reply:
note: IMHO call terms 21 and 22 is a bad practice

Reply:
Yeah, agreed